spec/ttml1.xml

```diff
@@ -3755,9 +3755,8 @@ represented by this attribute are based upon that defined by <bibref ref="xsl11"
 </div3>
 <div3 id="style-attribute-direction">
 <head>tts:direction</head>
-<p>The <att>tts:direction</att> attribute is used to specify a style property that
-defines the directionality of an embedding or override according to
-the Unicode bidirectional algorithm.</p>
+<p>The <att>tts:direction</att> attribute is used to specify a style property that, depending on the context of use,
+determines (1) the bidirectional paragraph level, or (2) the directionality of a bidirectional embedding or override.</p>
```

Comment:
There's an extra clause in https://github.com/w3c/ttml2/pull/562/files#diff-5ac39ab39265717d36dfd789d254ce1aR8859 that should probably be brought in here.

Reply:
@nigelmegitt I was not happy with the English "about which see". It seemed easiest to remove since UAX9 is already referenced below.

Reply:
@palemieux I don't see anything wrong with "about which see", but I see what you mean, and I agree the clause is not essential.